docs/configuration.md

```diff
@@ -1481,6 +1481,17 @@ Apart from these, the following properties are also available, and may be useful
     How many batches the Spark Streaming UI and status APIs remember before garbage collecting.
   </td>
 </tr>
+<tr>
+  <td><code>spark.streaming.listeners</code></td>
+  <td>(none)</td>
+  <td>
+    A comma-separated list of classes that implement <code>StreamingListener</code>; when initializing
+    SparkContext, instances of these classes will be created and registered with Spark's listener
+    bus.  If a class has a single-argument constructor that accepts a SparkConf, that constructor
+    will be called; otherwise, a zero-argument constructor will be called. If no valid constructor
+    can be found, the SparkContext creation will fail with an exception.
```

Comment:
Please update the documentation CORRECTLY. There are references to spark listener!!